routers/userRouting.js

```diff
@@ -49,4 +50,46 @@ router.get(`/logout/:username`, async ctx => {
   return au.logout(data);
 });
 
+router.post(`/:username/addCar`, async ctx => {
+  const data = {
+    username: ctx.params,
+    bodyParsed: ctx.request.body,
+    accessToken: ctx.request.headers.authorization,
+  };
+  if (await au.verifyToken(data.accessToken, data.username))
+    return car.addingCar(data);
+  else throw new Error('Verification false');
+});
+
+router.get(`/:username/cars`, async ctx => {
+  const data = {
+    username: ctx.params,
+    accessToken: ctx.request.headers.authorization,
+  };
+  if (await au.verifyToken(data.accessToken, data.username))
+    return car.userCars(data.username);
+  else throw new Error('Verification false');
+});
+
+router.post(`/:username/deleteCar`, async ctx => {
+  const data = {
+    username: ctx.params,
+    accessToken: ctx.request.headers.authorization,
+    carName: ctx.request.body,
+  };
+  if (await au.verifyToken(data.accessToken, data.username))
+    return car.deleteCar(data);
+  else throw new Error('Verification false');
+});
+
+router.post(`/:username/changeCar`, async ctx => {
```

Comment:
Every user can change any user data, you don't have authorization here

Reply:
It means, that anyone can change another ones cars?